docs/mobile.md

```diff
@@ -82,6 +82,10 @@ These instructions work only if your development computer is an Apple product. Y
 
 5. Revisit the Safari **Develop** menu of your laptop. You'll see a menu item that wasn't there before: the name of your phone.
 
-6. Within that menu item, you'll find a list of Safari browser tabs open on your iPhone. To see them, the iPhone must be unlocked, and the Safari browser must be displayed on the phone's screen.
+6. In that menu item, click the "Connect via Network" option. Once you've done this you should be able to unplug the USB cable and continue with these steps. A screenshot:
```

Comment:
Do you need "Connect via Network" to connect over USB cable like the rest of the instructions indicate? I'd assume connecting over the network would only be needed if you wanted to do some wireless debugging? (I'm not positive about this)

Reply:
I updated the docs to say more about this ([commit](https://github.com/18F/identity-idp/pull/9654/commits/8c8633c4535f6fec9ec9288eb675aef9215f233a)) and answer your question. Agreed that answering the "why" is important here.

That said, I think that everyone who follows these directions should check this box. It sets you free from the combination of USBDongle+ThunderboltCable+PluggedInPhone . A minor thing, but small annoyances can really add up for manual testing.

Reply:
> That said, I think that everyone who follows these directions should check this box. It sets you free from the combination of USBDongle+ThunderboltCable+PluggedInPhone . A minor thing, but small annoyances can really add up for manual testing.

My only question / concern might be if this may inadvertently open the developer up to exposing themselves if they're connected to a public WiFi network? Would it be correct to say this should only affect the options that the developer sees when choosing a tab to troubleshoot from within Safari, but doesn't affect whether their own device would be made available to others?